build/pdp10-ks/boot

```diff
@@ -3,6 +3,10 @@ set cpu its
 set cpu idle
 set dz 8b lines=8
 at -u dz0 10004
+set ch enabled
+set ch node=177002
```

Comment:
Ditto.  Also, for all the other hardcoded chaosnet and bridge addresses that follow in this PR.

Reply:
I'm going to go ahead with this.  If you want to propose a better way later, please do so!